dashboard/test/ui/features/projectsLegacy.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Projects
+
+# TODO - this isnt going to work in other environments (where this channel_id
```

Comment:
can you create the project in the test?
you can also run any arbitrary dashboard code (in any environment that works with tag @dashboard_db_access) if it's too cumbersome to do that through selenium